monai/bundle/scripts.py

```diff
@@ -202,7 +200,13 @@ def download(
 
     _log_input_summary(tag="download", args=_args)
     name_, bundle_dir_, source_, repo_, url_, progress_ = _pop_args(
-        _args, name=None, bundle_dir=None, source="github", repo=None, url=None, progress=True
+        _args,
+        name=None,
+        bundle_dir=None,
+        source="github",
+        repo="Project-MONAI/model-zoo/hosting_storage_v1",
```

Comment:
I think maybe no need to set default here as you already set in the `__init__()`?

Thanks.